doc/content/3.documentation/5.configuration/1.sagas/2.event.md

```diff
@@ -15,7 +15,7 @@ public record SubmitOrder(Guid OrderId);
 public class OrderStateMachine :
     MassTransitStateMachine<OrderState>
 {
-    public Event<SubmitOrder> SubmitOrder { get; private set; }
+    public Event<SubmitOrder> SubmitOrder { get; private set; } = null!;
```

Comment:
So, this is just noise, and I'd rather not force it. It isn't required, I use `#pragma` to disable the warnings.